server/index.js

```diff
@@ -43,10 +50,19 @@ class StatusRepository {
     return 'N/A'
   }
 
-  getStatus() {
-    const workingVotes = this.votes.count({ status: true, timestamp: { $gte: oneHourAgo() } })
-    const notWorkingVotes = this.votes.count({ status: false, timestamp: { $gte: oneHourAgo() } })
-    this.votes.findAndRemove({ timestamp: { $lt: oneHourAgo() } })
+  getStatus(parkKey) {
+    const workingVotes = this.votes.count({ status: true, timestamp: { $gte: oneHourAgo() }, parkKey })
```

Comment:
if one hour has passed will assume the splash pad is working if the last vote was working. Otherwise state unknown.